pkg/apiserver/user/auth.go

```diff
@@ -88,8 +91,9 @@ func NewAuthService() *AuthService {
 	}
 
 	service := &AuthService{
-		middleware:     nil,
-		authenticators: map[utils.AuthType]Authenticator{},
+		FeatureFlagNonRootLogin: featureFlagRegistry.Register("nonRootLogin", ">= 5.3.0"),
```

Comment:
Seems that this FeatureFlagNonRootLogin is not used. It doesn't look good as this may imply that this protection is purely happening at frontend and there is no backend protection for corresponding features.

Reply:
Create a new PR #1089 to refactor the "non root login" for better review experience.